sebs/cloudflare/cloudflare.py

```
@@ -0,0 +1,1608 @@
+        http_trigger = HTTPTrigger(func_name, worker_url)
+        http_trigger.logging_handlers = self.logging_handlers
+        worker.add_trigger(http_trigger)
+
+        return worker
+
+    def _get_worker(self, worker_name: str, account_id: str) -> Optional[dict]:
+        """Get information about an existing worker."""
+        headers = self._get_auth_headers()
+        url = f"{self._api_base_url}/accounts/{account_id}/workers/scripts/{worker_name}"
+
+        response = requests.get(url, headers=headers)
```

Ruff (0.14.10)

Probable use of `requests` call without timeout

(S113)